README.md

```diff
@@ -260,19 +261,45 @@ It is also straightforward to have a UIKit controller driven off of this store.
 Once we are ready to display this view, for example in the app's entry point, we can construct a store. This can be done by specifying the initial state to start the application in, as well as the reducer that will power the application:
 
 ```swift
+import ComposableArchitecture
+
 @main
 struct MyApp: App {
   var body: some Scene {
-    FeatureView(
-      store: Store(
-        initialState: Feature.State(),
-        reducer: Feature()
+    WindowGroup {
+      FeatureView(
+        store: Store(
+          initialState: Feature.State(),
+          reducer: Feature()
+        )
       )
-    )
+    }
   }
 }
 ```
 
+You may also find that the HTTP API request to numbersapi.com is blocked as only HTTPS connections are permitted.  If this happens you can add an exception to the project settings to allow HTTP requests just for the numbersapi.com domain.
```

Comment:
This seems to be a lot of unrelated information to display inline in a `README`. If it stays in the final version of this PR, it would probably be better to be hidden in a `details/summary` tag:
```html
<details>
  <summary>
  About `http://numbersapi.com`
  </summary>
  You may also find that the HTTP API request to […]
</details>
```
But I think it can probably go away, so please wait for other's feedback before making drastic changes.

Reply:
I agree, let this go which just leaves minor changes to using the samples in a cut/paste fashion